src/tools/qt-gui/src/treeviewmodel.cpp

```diff
@@ -562,7 +515,7 @@ void TreeViewModel::synchronize()
 
 	try
 	{
-		kdb.set(keySet, "/");
+		kdb.set(keySet, "/user");
```

Comment:
That should be "user". But I have to admit a good tutorial for namespaces is missing!
Please also mark that as TODO because obviously we want go back to "/" later.